package KillRankConfig;
import "RewardConfig.proto";

message KillRankItem
{
	repeated uint32 exp_killrange			= 1;   //经验杀敌数范围区间
	repeated float exp_coefficient			= 2;   //经验杀敌对应的系数
	repeated RewardConfig.Reward reward		= 3;   //排名奖励
}

message KillRank
{
	required KillRankItem killrank	= 1;
}
